boot/zephyr/main.c

```diff
@@ -44,6 +44,28 @@ const struct boot_uart_funcs boot_funcs = {
 #include <usb/class/usb_dfu.h>
 #endif
 
+#if defined(CONFIG_LOG) && !defined(CONFIG_LOG_IMMEDIATE)
+#ifdef CONFIG_LOG_PROCESS_THREAD
+#warning "The log internal thread for log processing can't transfer the log"\
+         "well for MCUBoot."
+#else
+#include <logging/log_ctrl.h>
+
+#define BOOT_LOG_STACK_SIZE 768
```

Comment:
I think @hakonfam 's comment about making this Kconfigurable (and the log processing interval too) is not addressed.

Reply:
> I think @hakonfam 's comment about making this Kconfigurable (and the log processing interval too) is not addressed.

Do you intend to address this one?

Reply:
addressed - I followed log internal thread stack size from zephyr codebase